web-client/src/modules/create/components/PostDetails.tsx

```diff
@@ -0,0 +1,170 @@
+import { ArrowRightOutlined, CloseOutlined } from '@ant-design/icons';
+import { Button, Form, Input } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import TitleWithOrangeUnderline from 'src/components/TitleWithOrangeUnderline/TitleWithOrangeUnderline';
+import { COLORS } from 'src/theme/colors';
+import styled from 'styled-components';
+
+const PostDetails: React.FC<PostDetailsProps> = ({
+  setPostDetails,
+  postDetails,
+  nextHandler: onNext,
+  prevHandler: handleCancel,
+}) => {
+  const { t } = useTranslation();
+  const [form] = Form.useForm();
+
+  const { title, type, body } = postDetails;
+
+  return (
+    <PostDetailsWrapper>
+      <FormWrapper>
+        <TitleWithOrangeUnderline level={2} color={COLORS.primaryDark}>
+          Post Details
+        </TitleWithOrangeUnderline>
+        <DetailsForm
+          layout="vertical"
+          form={form}
+          onFinish={values => {
+            setPostDetails(values);
+            onNext();
+          }}
+          initialValues={{ title, type, body }}
+        >
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'column',
+              justifyContent: 'space-between',
+              alignContent: 'space-between',
+            }}
+          >
+            <Form.Item
+              label="Title"
+              name="title"
+              rules={[
+                {
+                  required: true,
+                  message: t('newRequest.form.type_error_message'),
+                },
+              ]}
+            >
+              <Input />
+            </Form.Item>
+
+            <Form.Item
+              label={t('newRequest.form.type')}
+              name="type"
```

Comment:
Just to confirm, can this be any value or should it be a dropdown? behavior on development is a dropdown I believe

Reply:
Fixed to Dropdown